images/openwisp_openvpn/openvpn_utils.sh

```
@@ -0,0 +1,180 @@
+
+	mv "$tmp_checksum" checksum || return 1
+	rm -f "$tmp_tar"
+	rm -rf "$tmp_extract_dir"
+	trap - EXIT HUP INT TERM
+}
+
+function crl_download_to {
+	local output_path="${1:-revoked.crl}"
+	curl --silent --show-error --fail --insecure --output "$output_path" \
+	${DASHBOARD_INTERNAL}/admin/pki/ca/x509/ca/${CA_UUID}.crl
+}
```

Shellcheck (0.11.0)

[warning] 'function' keyword is non-standard. Use 'foo()' instead of 'function foo'.

(SC2113)